### DISCUSSION TITLE
Will it be possible to use em instead of px for font size?

### DISCUSSION THREAD

**Discussion by jrohlandt** [facebook/lexical#2932] [Q&A] [answered]
I'm working on project where I need to display the content of the text editor as a smaller preview.
What I'm currently doing with Draftjs is I convert the text editor content to HTML and then use js to manually scale down the font sizes (and line height, word spacing etc).
I'm not getting very accurate results with that approach but I'm sure if I spend countless more hours on it I can get it to an acceptable level.

However I'm thinking if I the rich text editor was able to use em instead of px for the font sizes (and anything else that uses px). Then all I would have to do for the preview is to scale down the base font of the editor accordingly and then everything else would just be correct?

Is that a good idea and is/will it be possible with Lexical?

**Comment by thegreatercurve** [comment, marked as answer]
It seems like a sensible way of creating a preview, and yes, this should be possible with Lexical. 

You have full access to the `theme` of the editor. There are no inherent CSS styles, and you can size the different node types as you see fit, including using `font-size`.

**Reply by jrohlandt** [reply]
That's good news. Thanks
